examples/example_emscripten_wgpu/CMakePresets.json

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
Not sure if this is very useful since it assumes many things (e.g. Ninja being used, MSVC being used etc.). People who use CMake are more likely to use it as-is without this file.

Reply:
Unfortunately this has been the main way I have been exposed to cmake, I had assumed if someone needed a preset they could add it to the list. As you pointed out I should have just put every step in the cmake file as they would allow people to choose how they compile it